src/singletons/Settings.hpp

```diff
@@ -251,6 +251,20 @@ class Settings : public ABSettings, public ConcurrentSettings
     QStringSetting selfHighlightColor = {"/highlighting/selfHighlightColor",
                                          ""};
 
+    BoolSetting enableSelfMessagesHighlight = {
+        "/highlighting/selfMessagesHighlight/nameIsHighlightKeyword", false};
+    BoolSetting showSelfMessagesHighlightInMentions = {
+        "/highlighting/selfMessagesHighlight/showSelfMessagesInMentions",
+        false};
+    //    BoolSetting enableSelfMessagesHighlightSound = {
+    //        "/highlighting/selfMessagesHighlight/enableSound", true};
+    //    BoolSetting enableSelfMessagesHighlightTaskbar = {
+    //        "/highlighting/selfMessagesHighlight/enableTaskbarFlashing", true};
+    //    QStringSetting selfMessagesHighlightSoundUrl = {
+    //        "/highlighting/selfMessagesHighlight", ""};
```

Comment:
Remove unused settings

Also want to revisit/check on the naming of these settings.